src/display/touch_manager.js

```diff
@@ -78,26 +80,61 @@ class TouchManager {
   }
 
   #onTouchStart(evt) {
-    if (this.#isPinchingDisabled?.() || evt.touches.length < 2) {
+    if (this.#isPinchingDisabled?.()) {
+      return;
+    }
+
+    if (evt.touches.length === 1) {
+      const pointerDownAC = (this.#pointerDownAC = new AbortController());
+      const signal = AbortSignal.any([this.#signal, pointerDownAC.signal]);
+      const container = this.#container;
+
+      // We want to have the events at the capture phase to make sure we can
+      // cancel them.
+      const opts = { capture: true, signal, passive: false };
+      const cancelPointerDown = e => {
+        if (e.pointerType === "touch") {
+          this.#pointerDownAC.abort();
```

Comment:
It probably doesn't matter, but let's prevent any possible future bugs by ensuring that it's still available here.
```suggestion
          this.#pointerDownAC?.abort();
```